tendrl/gluster_integration/message/callback.py

```diff
@@ -0,0 +1,12 @@
+class Callback(object):
+    def peer_connect(self, event):
+        pass
+
+    def peer_disconnect(self, event):
+        pass
+
+    def volume_create(self, event):
+        pass
+
+    def volume_delete(self, event):
+        pass
```

Comment:
Add entries for brick add/delete as well

Reply:
Sure, The 4 events were just sample callback functions added as part of framework, to give an idea how and where these functions will be defined. I will add more as part of successive patches

Reply:
Ack